lib/index.js

```diff
@@ -122,16 +121,12 @@ function TAccountsStrategy(options, verify) {
       // see normalized profile http://passportjs.org/docs/profile
       var normalizedProfile = {
         provider: 'taccounts',
-        id: body.userId,
-        displayName: body.profile.fullname
+        id: body.sub,
+        displayName: body.name
       };
 
-      normalizedProfile.emails = body.identities.emails.map(function(email) {
-        return {
-          value: email.address,
-          primary: email.main
-        };
-      });
+      normalizedProfile.emails = [];
+      normalizedProfile.emails.push(body.email);
```

Comment:
I think you have to do:

```
normalizedProfile.emails.push({value: body.email, primary: true});
```

because that's what passport expect in a passport normalized profile.